03. 문자열 탐색/02. 유효한 팰린드롬/이서인/index.js

```diff
@@ -1,8 +1,17 @@
 function solution(s) {
-  let answer = 'YES';
+  const alphabet = Array.from({ length: 26 }, (v, i) =>
+    String.fromCharCode(i + 97)
+  );
+  Array(26)
+    .fill()
+    .map((v, i) => String.fromCharCode(i + 97));
 
-  return answer;
+  const filteredS = [...s.toLowerCase()]
+    .filter((el) => alphabet.includes(el))
```

Comment:
아스키 코드를 봐서 반대로 문자를 아스키코드로 바꾸는 방법도 떠오르네요~

```suggestion
  const filteredS = [...s.toLowerCase()]
    .filter((el) => el.charCodeAt() >= 97 && e.charCodeAt() <= 122)
```